starters/vue3-xstate-css/src/App.vue

```diff
@@ -0,0 +1,3 @@
+<template>
+	<RouterView />
```

Comment:
Nice catch - the prettier file is actually set to use tabs instead of spaces, but the editor config never got updated along with that. So I'll fix this in the editor config!